tests/test_integration/test_run_transfer/test_hdfs.py

```diff
@@ -164,6 +172,12 @@ async def test_run_transfer_hdfs_to_postgres(
         table="public.target_table",
     )
     df = reader.run()
+
+    # as initial datetime values in excel are rounded to milliseconds
```

Comment:
According to spark-excel README, default value for `timestampFormat` is `yyyy-mm-dd hh:mm:ss[.fffffffff]`, so it shouldn't round anything. Probably the original .xlsx file contains rounded value in the first place

Reply:
Yes, original excel file does not support higher precision than milliseconds. Comment is trying to say that

Reply:
Is it possible to create any Excel file with timestamp precision more than a millisecond?

Reply:
It is not directly supported at the moment. Theoretically, we can store strings, but this option does not seem intuitive or native for users

Reply:
If the format itself doesn't support that, we can just update comments to mention that. No need to make implicit type conversions here